packages/inter-protocol/src/price/priceAggregatorChainlink.js

```diff
@@ -313,9 +309,27 @@ export const start = async (zcf, privateArgs, baggage) => {
     getPriceAuthority() {
```

Comment:
Does this Far object need to be updated at this time?

Reply:
not until https://github.com/Agoric/agoric-sdk/issues/6853